Tests/RemoveKnownDuplicatesTests.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  RemoveKnownDuplicatesTests.swift
+//  CombineExtTests
+//
+//  Created by Jasdev Singh on 21/03/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+import CombineExt
+import XCTest
+
+final class RemoveKnownDuplicatesTests: XCTestCase {
+    private var subscription: AnyCancellable!
+
+    private enum RemoveKnownDuplicatesTestError: Error {
+        case anError
+    }
+
+    // MARK: - `Hashable`-related tests
+
+    func testHashableExpectedDeduplication() {
+        var results = [Int]()
+
+        subscription = [1, 1, 2, 1, 3, 3, 4].publisher
+            .removeKnownDuplicates()
```

Comment:
Maybe create `Hashable`-conforming and `Equatable`-conforming structs separately here?